contrib/fasttext/template.py

```diff
@@ -0,0 +1,22 @@
+pkgname = "fasttext"
+pkgver = "0.9.2"
+pkgrel = 0
+build_style = "cmake"
+hostmakedepends = ["cmake", "ninja", "pkgconf"]
+pkgdesc = "Library for fast text representation and classification"
+maintainer = "Duncan Bellamy <dunk@denkimushi.com>"
+license = "MIT"
+url = "https://fasttext.cc/index.html"
+source = f"https://github.com/facebookresearch/fasttext/archive/refs/tags/v{pkgver}.zip"
+sha256 = "93cc8a21633e6c6ecf4b5164925ad68a5eda332373d4316d2f56d05f4bfbe823"
+# Tests require 300+MB test data
+# options = ["!check"]
```

Comment:
unwanted comment

Reply:
any reason you dropped the whole thing? is the test data comment not correct?

Reply:
I thought that’s what you meant as without the comment it fails locally, comment is correct.  It has it’s own script to download test data: https://github.com/facebookresearch/fastText/blob/main/tests/fetch_test_data.sh

Reply:
what? the `!check` is commented out, so it doesn't do anything

Reply:
ah, I see. Missed that